views/graph/show.handlebars

```diff
@@ -33,6 +33,11 @@
             Share
         </button>
 	</div>
+	<h2 id="graph_heading">
```

Comment:
ermm.. this shouldn't be here? there is one up above already. (assuming this is based on latest develop)